### ISSUE TITLE
Templating support in helidon static handlers

### ISSUE THREAD

**Issue by gcameo** [helidon-io/helidon#220]
Is there a way to provide support for some form of templating in Helidon. At the moment, only static html can be rendered. Will be nice to support things like MVEL, Freemarker or some other templating framework so we can have a bit of dynamic content

**Comment by spericas** [comment]
At least for MP, we could consider integration with Ozark (https://www.mvc-spec.org/).

**Comment by romain-grecourt** [comment]
We should also able to support some templating in helidon-se with mustach or freemarker.

**Comment by gcameo** [comment]
https://github.com/fizzed/rocker

**Comment by romain-grecourt** [comment]
Ok, or rocker like @gcameo suggests ;)

**Comment by carimura** [comment]
i know this is slightly unrelated to the issue but just for posterity I found a workaround template system that works nicely (and easily) without the need for this ticket to come out of backlog: https://github.com/HubSpot/jinjava

**Comment by anthonyvdotbe** [comment]
Integration with Eclipse Krazo would be a good start. A comeback of [String Templates](https://openjdk.org/jeps/465) would also be useful in this context.
For comparison, Quarkus has pretty extensive [support for web applications](https://quarkus.io/guides/web).

**Comment by dasTanmoy0096** [comment]
Any plans for this?

**Comment by windscale-stephen** [comment]
I agree templating support for HTML generation would be good. Is there a Jakarta-EE or Microprofile standard that would make sense?
